test/demo/createNotification.inttest.ts

```diff
@@ -8,7 +8,7 @@ import * as notifsRemote from './utils/notificationsRemoteExecution';
 import {getCardsByNotification, isCenterShowing} from './utils/notificationCenterUtils';
 import {delay} from './utils/delay';
 import {getToastWindow, getToastCards} from './utils/toastUtils';
-import {createTestApp} from './utils/createTestApp';
+import * as spawnRemote from './utils/spawnRemote';
```

Comment:
Why the `*` import here? Would `await createApp` not be more readable within the tests?

Reply:
Quite possibly. It's just how I did it at the time in line with the notifsRemote that I'm using in many more places. I'm not super fussed either way, but I do at least like that this makes it clear that it's a remote window call as opposed to a local one.

Reply:
Either way, changed it to a named import for now. Can change again if anyone has any stronger opinions.